go.mod

```
@@ -1,9 +1,8 @@
 module github.com/chainguard-dev/dfc
 
-go 1.23.6
+go 1.24
```

```suggestion
go 1.23.6
```

see https://github.com/chainguard-dev/dfc/issues/2

---

i think he should upgrade to go1.24 and not use go1.23 :)

but I can revert that